### DISCUSSION TITLE
Failed at the @hyas/doks@0.4.2 postinstall script.

### DISCUSSION THREAD

**Discussion by fleurien** [thuliteio/doks#723] [Q&A]
Hello! I am new to Doks and jamstack in general. I'm trying to get Doks set up but I have run into this error when doing the `npm install` command:

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @hyas/doks@0.4.2 postinstall: `hugo-installer --version otherDependencies.hugo --extended --destination node_modules/.bin/hugo
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @hyas/doks@0.4.2 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Nem\AppData\Roaming\npm-cache\_logs\2022-04-09T17_44_16_100Z-debug.log

Here is the result from the `npm run check` which also generated an error:

> doks-child-theme@0.4.2 precheck D:\repos\nem-docs
> npm version

{ 'doks-child-theme': '0.4.2',
  npm: '6.9.0',
  ares: '1.15.0',
  brotli: '1.0.7',
  cldr: '35.1',
  http_parser: '2.8.0',
  icu: '64.2',
  modules: '64',
  napi: '4',
  nghttp2: '1.34.0',
  node: '10.16.0',
  openssl: '1.1.1b',
  tz: '2019a',
  unicode: '12.1',
  uv: '1.28.0',
  v8: '6.8.275.32-node.52',
  zlib: '1.2.11' }

> doks-child-theme@0.4.2 check D:\repos\nem-docs
> exec-bin node_modules/.bin/hugo/hugo version

D:\repos\nem-docs\node_modules\exec-bin\bin\exec-bin.js:2
import { spawn } from 'child_process';
       ^

SyntaxError: Unexpected token {
    at Module._compile (internal/modules/cjs/loader.js:721:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! doks-child-theme@0.4.2 check: `exec-bin node_modules/.bin/hugo/hugo version
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the doks-child-theme@0.4.2 check script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Nem\AppData\Roaming\npm-cache\_logs\2022-04-09T17_45_14_904Z-debug.log`

Here is a link to the repo: https://github.com/nemesyzygy/nem-docs 

Much appreciated!

**Comment by h-enk** [comment]
Please upgrade [Node.js](https://nodejs.org/en/) to the latest LTS version or newer.
